.github/workflows/helm.yml

```diff
@@ -125,12 +127,41 @@ jobs:
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
+      - name: Install cosign
+        uses: sigstore/cosign-installer@v3.7.0
+        with:
+          cosign-release: 'v2.4.1'
+
       - name: Push chart to GHCR
+        id: push_chart
         run: |
           shopt -s nullglob
           for pkg in .cr-release-packages/*.tgz; do
             if [ -z "${pkg:-}" ]; then
               break
             fi
-            helm push "${pkg}" "oci://ghcr.io/${GITHUB_REPOSITORY_OWNER}/charts"
-          done
\ No newline at end of file
+            chart_name=$(helm show chart "${pkg}" | yq .name)
+            # helm push fails when registry path contains Uppercase letters
+            chart_registry="ghcr.io/$(echo $GITHUB_REPOSITORY_OWNER |  tr '[:upper:]' '[:lower:]')/charts"
```

Comment:
This isn't needed since the repository owner is `external-secrets`. And we someone forks external secrets they can fix their own helm action anyways.

Reply:
Hey, that makes sense. I have removed this change.